jenny/templates/jobs_collect_static.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.django.do_collect_static }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "jenny.fullname" . }}-collectstatic
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "jenny.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{ toYaml .Values.commonLabels | nindent 4 }}
+    {{- end }}
+    app.kubernetes.io/component: jenny
+  annotations:
+{{- if .Values.commonAnnotations }}
+{{- toYaml .Values.commonAnnotations | nindent 4 }}
+{{- end }}
+    "helm.sh/hook": pre-upgrade, pre-install
+    "helm.sh/hook-weight": "0"
+spec:
+  template:
+    metadata:
+      labels: {{- include "jenny.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: django
+    spec:
+      {{- include "django.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.podSecurityContext }}
+      securityContext: {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: django-collectstatic
+          image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- include "jenny.envs" . | nindent 10 }}
+          command: ["python", "manage.py", "collectstatic", "--no-input"]
+          {{- if .Values.resources }}
+          resources: {{ toYaml .Values.resources | nindent 12 }}
+          {{- end }}
```

Comment:
If persistent is enabled, this job should probably use the static PVC and use a volume

Reply:
resolved by https://github.com/openfun/charts/pull/4/commits/dfe2f48466b4c6dcfb3e97509277ed9c7b85137a